website/static/js/pages/base-page.js

```diff
@@ -58,6 +58,11 @@ var SlideInViewModel = function (){
     };
     // Google Analytics click event tracking
     self.trackClick = function(source) {
+        if (source === 'Create Account') {
+            window.location = '/#signUpScopeTop';
```

Comment:
This link will be visible to the user. Let's make the ID more concise. `/#signUp` , perhaps.